packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -364,3 +364,31 @@ export class CCloseVault extends ComposableBuffer<CloseVault> {
     ]
   }
 }
+
+/**
+ * PlaceAuctionBid DeFi Transaction
+ */
+export interface PlaceAuctionBid {
+  vaultId: string // ------------------| 32 bytes, Vault Id
+  index: string // --------------------| 4 bytes, Auction batches index
```

Comment:
index should be `number` type to tally with the cpp side data structure.
```
struct CAuctionBidMessage {
    CVaultId vaultId;
    uint32_t index;
    CScript from;
    CTokenAmount amount;

    ADD_SERIALIZE_METHODS;

    template <typename Stream, typename Operation>
    inline void SerializationOp(Stream& s, Operation ser_action)
    {
        READWRITE(vaultId);
        READWRITE(index);
        READWRITE(from);
        READWRITE(amount);
    }
};
```
it fixes https://github.com/DeFiCh/jellyfish/pull/722/files#r748120466 as well